pom.xml

```diff
@@ -219,6 +219,7 @@
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
       <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
+      <maven.test.failure.ignore>false</maven.test.failure.ignore>
       <maven.test.redirectTestOutputToFile>false</maven.test.redirectTestOutputToFile>
```

Comment:
I would make this one true.. and anyone else actually requiring to not generate it, should make it false.

The other day I was having a hard time figuring out how to get the output of my tests... and I didn't know why it was not generated..


If I, working on this codebase daily, had a difficult time... imaging other people.

Reply:
not related to your changes I know, but since you are there.. can you make it true please?

Reply:
I can, and have always wanted to...that prop is only even there as I wanted to be able to, so I changed it so allow configuring with the standard prop because it was hard coded and thus not configurable at all originally: https://github.com/apache/activemq-artemis/commit/0cdd0c8bd9050e203799cc0cbfa3f1d055fb7b26

However, not in this PR, since it also needs adjustments made to CI jobs to either flip it back, or begin actually archiving the results files so you can ever see the output at all.